observability/micrometer/src/main/java/org/wildfly/extension/micrometer/model/MicrometerSchema.java

```diff
@@ -0,0 +1,31 @@
+package org.wildfly.extension.micrometer.model;
+
+import java.util.Locale;
+
+public enum MicrometerSchema {
+    VERSION_1_0(1, 0), // WildFly 26
```

Comment:
Minor note, we should either drop this comment or remember to change it before it's merged.

Reply:
We'll fix that in the final release.